backend/schema/corpora_schema.md

```diff
@@ -0,0 +1,331 @@
+and users. So while we might be strict about our validation policies at any given time, our software needs to be very flexible. We need to avoid a situation
+where we can't evolve schema and curation policies because it requires expensive software updates. There will be situations where software features or
+capabilities will be hard or impossible because we prioritize this flexibility.
+
+## Schema
+
+### Original Dataset Requirements
+
+
+These are the fundamental requirements that all dataset must meet, the handful of things our software depends on and mandatory legal requirements. All
+submitted data must comply with these requirements.
+
+
+*   **Unique observation identifiers**. Each observation (usually a cell) must have an id that is unique within the dataset. This is usually already
+    present in every submission, for example as a barcode.
+*   **Unique feature identifiers**. Every feature (usually a gene or transcript) also needs a unique identifier. This is occasionally not present because
+    of one-to-many mappings between gene symbols and other gene ids. In cases where there are duplicated feature identifiers, they will be summed during
+    curation.
+*   **Type annotations or inferrable types**. For both expression values and metadata, we don't want to have to guess at the type. Typing is already explicit
+    in anndata, loom, Seurat and Bioconductor formats.
+*   **No PII**. No metadata can be personally identifiable: no names, dates of birth, specific locations, etc. There's a
+    [list](https://docs.google.com/document/d/1nlUuRiQ8Awo_QsiVFi8OWdhGONjeaXE32z5GblVCfcI/edit?usp=sharing).
+
+
+#### Matrix Layers
+
+The count matrix itself can exist in several forms. Generally, there is a "raw" count matrix without scaling, filtering, normalization, etc. And there is
+a "final" matrix that is used in the publication with QC and various corrections. There can also be several intermediate matrices.
+
+These different transformations of the count matrix are called "layers" in several file formats. Submissions to Corpora must contain at least one layer,
+the "raw" layer. They may optionally also have a "final" layer used for presentation, and each layer must be identified.
+
+
+### Remix Metadata
+
+The remaining part of the schema defines what's needed for the Corpora remix. The words "must" and "requirement" are used throughout, but note the
+discussion above regarding curation policies as well the [caution against](#Software-vs-policy) introducing any
+software dependencies on these fields.
+
+
+#### Integration Metadata
+
+To support data integration, each cell must have the following metadata:
+
+**Field name**|**Constraints**
+:--|:--
+tissue|string (see note below regarding cell culture and organoids)
+assay|string
+disease|string
+cell\_type|string
+sex|"male", "female", "mixed", "unknown"
```

Comment:
is this sufficient for non-mammalian data?

Reply:
No, though we don't have any of that. I can add back "other".